packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -32,6 +32,17 @@ export default class QuanticCategoryFacet extends LightningElement {
   @api engineId;
   /** @type {boolean} */
   @api noSearch = false;
+  /** @type {string} */
+  @api basePath = '';
+  /** @type {boolean} */
+  @api noFilterByBasePath = false;
```

Comment:
do we support `filterFacetCount` for categoryFacet?

Reply:
why not.